src/Inflector/Inflector.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Inflector;
+
+use Doctrine\Common\Inflector\Inflector as LegacyInflector;
+use Symfony\Component\Inflector\Inflector as SymfonyInflector;
+
+/**
+ * @internal
+ *
+ * @deprecated removed in version 3.0
+ */
+final class Inflector
+{
+    public static function tableize(string $word): string
+    {
+        if (\function_exists('\Symfony\Component\String\u')) {
+            return (string) \Symfony\Component\String\u($word)->snake();
+        }
+        @trigger_error('This method is deprecated use symfony/string',
```

Comment:
```suggestion
        @trigger_error('Using the Doctrine inflector is deprecated since API Platform 2.7 and will not be supported anymore in 3.0, use the Symfony string component instead.',
```